### PR TITLE
[CLT-2022] Venue secured, add new event for Charlotte 2022

### PR DESCRIPTION
**Pull request by jmaggio**
We have a confirmed venue and dates and are adding the 2022 Charlotte event! Updates to come as we kick off the planning for this year but wanted to go ahead and get the event with just the dates and location up.

### CODE REVIEW

File: data/events/2022-charlotte.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+organizer_email: "charlotte@devopsdays.org" # Put your organizer email address here
+
+# List all of your sponsors here along with what level of sponsorship they have.
+# Check data/sponsors/ to use sponsors already added by others.
+sponsors:
+  - id: samplesponsorname
+    level: gold
+ #  url: http://mysponsor.com/?campaign=me # Use this if you need to over-ride a sponsor URL.
+  - id: arresteddevops
+    level: community
```
Comment (phrawzty) [inline data/events/2022-charlotte.yml:71]: This is the default template. You'll want to remove ADO if they're not a sponsor, for example (side-bar: that's something we should probably fix in the template, but I digress).

File: data/events/2022-charlotte.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2022-charlotte.yml:51]: I took a quick look at your existing mailing list and it seems to line up here, but if there are any changes, please let us know via email to info@devopsdays.org (w/ names and emails). 👍

File: data/events/2022-charlotte.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#
+location_address: "1101 Red Ventures Drive, Indian Land, SC 29707"
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
```
Comment (phrawzty) [inline data/events/2022-charlotte.yml:42]: This leads to the default sponsor template page, which might not be what you want. Please review and either confirm or modify (i.e. `#`) as appropriate.

File: data/events/2022-charlotte.yml (hunk 2 above)
Comment (jmaggio) [reply data/events/2022-charlotte.yml:51]: There are surely going to be some updates to come for our organizing team this year. I kept the same team for now and I will make updates to team_members / send the email as we get confirmation from each person by next week-ish.

File: data/events/2022-charlotte.yml (hunk 3 above)
Comment (jmaggio) [reply data/events/2022-charlotte.yml:42]: Yep that's actually what I was intended for now. Just launch the event w/ date / location for now and make the Charlotte sponsor page Charlotte specific in the near future.

**Pull request merged**